themes/classic/templates/catalog/_partials/product-variants.tpl

```diff
@@ -35,4 +35,5 @@
       {/if}
     </div>
   {/foreach}
+  <input type="hidden" name="idCombination" id="idCombination" value="{$product.id_product_attribute}">
```

Comment:
But what if I want to hide/show own blocks (generated by my module) right after customer will change attribute at the product page?
For now I don't have this ability.
For example I want to show block only for T-Shirt Green color / M size. But for another attributes block should be hidden. Default-bootstrap in Prestashop 1.6 have this "idCombination" hidden field.